### PR TITLE
adding testGas

### PR DESCRIPTION
**Pull request by AndreiD**
adding testGas & consistent " instead of '

### CODE REVIEW

File: contracts/ERC721A.sol [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 // Creator: Chiru Labs
 
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.11;
```
Comment (cygaar) [inline contracts/ERC721A.sol:4]: Let's keep this at 0.8.0 to maintain maximum compatibility

File: hardhat.config.js [hunk 2]
```diff
@@ -9,8 +9,18 @@ if (process.env.REPORT_GAS) {
  * @type import('hardhat/config').HardhatUserConfig
  */
 module.exports = {
-  solidity: "0.8.4",
```
Comment (cygaar) [inline hardhat.config.js:12]: Can you actually change this to 0.8.0? Missed this during my initial setup

**Review by cygaar — CHANGES_REQUESTED** [review]

**Comment by cygaar** [comment]
@AndreiD yeah I misspoke, it's already been changed in the main branch

**Comment by cygaar** [comment]
The contract pragma version should be changed though

**Comment by AndreiD** [comment]
to what? 0.8.0? but I would remove the ^..... some security auditors don't like it

**Comment by cygaar** [comment]
We want to match up with OpenZeppelin which uses ^0.8.0

**Review by cygaar — APPROVED** [review]

**Comment by cygaar** [comment]
@AndreiD can you handle the merge conflicts, then I can merge this?

**Comment by AndreiD** [comment]
why the package-lock file was there.... :( no idea.

**Comment by AndreiD** [comment]
check now

**Comment by cygaar** [comment]
Seems like there's still an issue - I think you can just checkout the package-lock from the main branch

**Comment by AndreiD** [comment]
close this one. I'll open a new pull request

**Pull request closed without merging**